.prettierrc.json

```diff
@@ -1,6 +1,5 @@
 {
   "singleQuote": true,
-  "trailingComma": "es5",
```

Comment:
Now the default: https://prettier.io/blog/2020/03/21/2.0.0.html#change-default-value-for-trailingcomma-to-es5-6963httpsgithubcomprettierprettierpull6963-by-fiskerhttpsgithubcomfisker